src/apps/search/GeosearchFilter.tsx

```diff
@@ -0,0 +1,33 @@
+import TranslationContext from '@apps/search/TranslationContext';
+import { Checkbox, useGeoSearchToggle } from '@performant-software/core-data';
+import { useRuntimeConfig } from '@peripleo/peripleo';
+import { useContext } from 'react';
+
+const GeosearchFilter = () => {
+  const config = useRuntimeConfig<any>();
+  const { filterByMapBounds, setFilterByMapBounds } = useGeoSearchToggle();
+  const { t } = useContext(TranslationContext);
+
+  if (!config.search?.geosearch) {
+    return null;
+  }
+
+  return (
+    <div
+      className='text-sm flex items-center'
+    >
+      <Checkbox
```

Comment:
After https://github.com/performant-software/react-components/pull/405, the Checkbox component expects an `ariaLabel` prop that it can pass down to its internal `<button>`. This is true even if we're already using a `<label>` because a11y checks don't consider a `<label>` as text for a `<button>` like it would for an `<input type="checkbox" />`.